docs/source/models/templates/concept_model_template.rst

```diff
@@ -72,188 +75,231 @@ Concept model template
 
-4.1 Simulation scenarios
-------------------------
+This section is intended as the main body of documentation of your simulation. Information
+in this section might include the following: 
 
+* Concept model diagram 
+* Links to relevant components of diagram
+* Scenario descriptions
+* Default specifications 
+* Demographics
+* Location description
+* Effect size
 
-.. _{YOUR_MODEL_SHORT_NAME}5.0:
+.. _{YOUR_MODEL_SHORT_NAME}2.1:
 
-5.0 Vivarium modelling components
-+++++++++++++++++++++++++++++++++
+2.1 Concept model diagram 
+-------------------------
 
-.. _{YOUR_MODEL_SHORT_NAME}5.1:
+.. _{YOUR_MODEL_SHORT_NAME}2.2:
 
-5.1 Vivarium concept model 
+2.2 Default specifications 
 --------------------------
 
-.. note::
-  This is our standard vivarium concept model diagram we are used to seeing
-
-.. _{YOUR_MODEL_SHORT_NAME}5.2:
+The below table is intended to outline the default specifications of your simulation. 
+Included in the table is a column of parameter definitions. Please delete this column as you 
+fill out the table. 
+
+.. list-table:: Default simulation specifications
+  :header-rows: 1
+
+  * - Parameter
+    - Definition
+    - Value
+    - Note
+  * - Location(s)
+    - Target/simulated region & GBD ID 
+    - e.g. Ethiopia (ID: 179)
+    -
+  * - Number of draws
+    - Desired number of draws that a given simulation is to be run for. (Generally, this should be a number between 1 and 999.)
+    - e.g. 50 draws 
+    - Read more about draws in Vivarium :ref:`here <vivarium_best_practices_monte_carlo_uncertainty>`.
+  * - Population size per draw
+    - Desired simulated population size per draw for a given simulation. 
+    - e.g. 100,000 simulants
+    - Read more about how to determine a reasonable population size for your simulation 
+      :ref:`here <vivarium_best_practices_monte_carlo_uncertainty>`.
+  * - Age start (initialization)
+    - If sim duration > age start for observation, age_start at initialization will be zero and
+      fertility will need to be included. Otherwise, age start at initialization will be age start for 
+      observation minus simulation duration and a closed cohort (no fertility).
+    - e.g. 0 months
+    - Currently vivarium can only have new simulants enter the simulation at age zero, although 
+      changing this could be a potential framework improvement. 
+  * - Age start (observation)
+    - Age at which simulants are included in observer.
+    - e.g. 6 months
+    -
+  * - Age end
+    - Age at which simulants are no longer included in observer. 
+    - e.g. 5 years
+    -
+  * - Exit age
+    - Age at which simulants age out of the simulation. 
+    - e.g. 5 years
+    -
+  * - Simulation start date
+    - 
+    - e.g. 2021-07-01
+    -
+  * - Simulation observation start date
+    - 
+    - e.g. 2022-01-01
+    -
+  * - Simulation end date
+    - 
+    - e.g. 2026-12-31
+    -
+  * - Timestep
+    - Amount of time that passes in simulation between each observer.
```

Comment:
Maybe this was copied from somewhere, but "between each observer" doesn't sound quite right to me -- there are other reasons to have a short time step that don't have to do with observation. Maybe:

```suggestion
    - Amount of time that passes in simulation between each "step," which is when events and observations can happen.
```